### DISCUSSION TITLE
Release 23.10: Number of PHY DL threads (i.e., 4) cannot be larger than the maximum processing delay in slots (i.e., 2).

### DISCUSSION THREAD

**Discussion by Zplusless** [srsran/srsRAN_Project#267] [General Help] [answered]
the previous master version (commit 5e6f50a20) works well, but the newly built release 23.10 failes to work.

![image](https://github.com/srsran/srsRAN_Project/assets/10062702/b51d613e-ae33-41f1-aeb3-8e8f22814cb2)

I'm using the default config file for usrp-b210 provided by release-23.10:

[gnb_rf_b210_fdd_srsUE_2310.yml.txt](https://github.com/srsran/srsRAN_Project/files/13070197/gnb_rf_b210_fdd_srsUE_2310.yml.txt)

**Comment by AlaiaL** [comment, marked as answer]
Hi Zplusless,

Thanks for spotting this bug and reporting. A hotfix is coming out soon.

**Reply by Zplusless** [reply]
Thanks, I compiled and installed previous master version (commit https://github.com/srsran/srsRAN_Project/commit/5e6f50a202c6efa671d5b231d7c911dc6c3d86ed) first. 

Then I compiled other release versions without remove the previous version. I tried release_23_5 and release_23_3, both failed to run with following message:

```bash
# error message for release_23_5 and release_23_3
INI was not able to parse ru_sdr.++
Run with --help for more information.
```

Then release 23.10 gives the error message mentioned [in the above comment](https://github.com/srsran/srsRAN_Project/discussions/267#discussion-5767922)

Is there any problem running multiple versions on the same server? Will the default configs interfere with each other？

**Reply by AlaiaL** [reply]
There is nor problem running multiple version on the same server. The error you are getting there is because the configuration file changed with the last release, as new features were added to the gNB.
You could try to run the release_23_5 or release_23_3 with the configuration files from their respective releases.

Also, the hotflix is already public here: 
https://github.com/srsran/srsRAN_Project/tree/release_23_10_1

so you can give it a try also with your current configuration.
